crates/swift-bridge-ir/src/codegen/generate_swift.rs

```diff
@@ -401,7 +401,7 @@ fn gen_func_swift_calls_rust(
     let swift_class_func_name = if function.is_swift_initializer {
         "convenience init".to_string()
     } else {
-        format!("func {}", fn_name.as_str())
+        format!("public func {}", fn_name.as_str())
```

Comment:
`cargo test -p swift-bridge-ir`

A bunch of tests will probably be failing. You just need to add the word `public to them and they should pass!